frontend/public/static/locale/en-US/translate.ftl

```diff
@@ -258,6 +258,12 @@ entitydetails-ContextIssueButton--context-issue-button = REQUEST CONTEXT or REPO
 entitydetails-GenericOriginalString--plural = PLURAL
 entitydetails-GenericOriginalString--singular = SINGULAR
 
+## Entities List
+## Show a list of entities
+
+entitieslist-Entity--title =
+    .title = Click to reveal surrounding strings
```

Comment:
Let's use the word sibling (instead of surrounding) here for consistency. Sorry for the confusion in previous review!